src/main/java/io/jenkins/plugins/analysis/warnings/PVSStudio.java

```diff
@@ -0,0 +1,75 @@
+package io.jenkins.plugins.analysis.warnings;
+
+import edu.hm.hafner.analysis.IssueParser;
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.jenkinsci.Symbol;
+import hudson.Extension;
+
+import io.jenkins.plugins.analysis.core.model.IconLabelProvider;
+import io.jenkins.plugins.analysis.core.model.ReportScanningTool;
+import io.jenkins.plugins.analysis.core.model.StaticAnalysisLabelProvider;
+
+import edu.hm.hafner.analysis.parser.pvsstudio.PVSStudioParser;
+
+/**
+ * Provides a parser and customized messages for PVS-Studio static analyzer.
+ *
+ * @author PVS-Studio Team
+ */
+public class PVSStudio extends ReportScanningTool {
+
+    private static final long serialVersionUID = -1114828406964963020L;
+    private static final String ID = "pvs-studio"; // history chart title
```

Comment:
Can you please add a license note for the icon to [LICENSE.txt](https://github.com/jenkinsci/warnings-ng-plugin/blob/master/src/main/webapp/icons/LICENSE.txt)?